camus-shopify/src/main/scala/com/linkedin/camus/shopify/CamusExecutions.scala

```diff
@@ -42,4 +47,25 @@ class CamusExecutions(properties: Properties, fs: FileSystem) {
     }
     list.toList
   }
+
+  def lastRunDirs(numRuns: Int): List[Path] = camusReader.mostRecentRuns(new Path(historyFolder), numRuns)
+
+  def droppedFoldersInRuns(camusRunPaths: List[Path]): Seq[String] = {
+    val allPathsValid = camusRunPaths.forall(path => fs.exists(new Path(path, EtlMultiOutputFormat.PATHS_WRITTEN_PREFIX)))
+    if (! allPathsValid) {
+      throw new Exception("Some executions paths do not contain expected dirs lists.")
```

Comment:
this means we'll have to run for a bit without this config (`check-type=runs`) on to seed `dirs-written-to`, right? (it's fine, just validating my understanding)

Reply:
Exactly!